ttnn/cpp/ttnn/operations/eltwise/binary/device/binary_device_operation.cpp

```diff
@@ -17,8 +17,8 @@ namespace ttnn::operations::binary {
 namespace utils {
     bool is_binary_sfpu_op(BinaryOpType val, DataType a, DataType b) {
     switch (val) {
-        case BinaryOpType::ADD: return ((a == DataType::FLOAT32 && b == DataType::FLOAT32) || (a == DataType::INT32 && b == DataType::INT32));
-        case BinaryOpType::SUB:
+        case BinaryOpType::ADD:
+        case BinaryOpType::SUB: return ((a == DataType::FLOAT32 && b == DataType::FLOAT32) || (a == DataType::INT32 && b == DataType::INT32));
```

Comment:
I had to read this diff like 4 times to finally see what the change was. Can we move the `return` to be on its own line after the last `case` that applies to it, rather than having it on the same line? Thanks.